nilearn/image/tests/test_resampling.py

```diff
@@ -1265,3 +1267,20 @@ def test_smoke_resampling_non_nifti(affine_eye, shape, rng, force_resample):
         force_resample=force_resample,
         copy_header=True,
     )
+
+
+@pytest.mark.parametrize("shape", [(1, 4, 4)])
+def test_get_mask_bounds(data, affine_eye):
+    img = Nifti1Image(data, affine_eye)
+    assert_allclose((0.0, 0.0, 0.0, 3.0, 0.0, 3.0), get_mask_bounds(img))
```

Comment:
@bthirion I've added tests. However I've used the results returned from the function.